pythonforandroid/pythonpackage.py

```diff
@@ -172,11 +173,22 @@ def _get_system_python_executable():
 
     def python_binary_from_folder(path):
         def binary_is_usable(python_bin):
+            """ Helper function to see if a given binary name refers
+                to a usable python interpreter binary
+            """
+
+            # Abort if path isn't present at all or a directory:
+            if not os.path.exists(os.path.join(path, python_bin)) or \
```

Comment:
**Minor suggestion:** I think that this is more readable this way and we avoid using backslash:
```
            if not os.path.exists(
                os.path.join(path, python_bin)
            ) or os.path.isdir(os.path.join(path, python_bin)):
                return
```

Reply:
added, thanks :heart: